lib/grape/router.rb

```diff
@@ -160,7 +169,7 @@ def greedy_match?(input)
     end
 
     def call_with_allow_headers(env, methods, endpoint)
-      env[Grape::Env::GRAPE_ALLOWED_METHODS] = methods.join(', ')
+      env[Grape::Env::GRAPE_ALLOWED_METHODS] = -methods.join(', ')
```

Comment:
explain?

Reply:
For those who are using ruby >= 2.5
https://rubyreferences.github.io/rubychanges/2.5.html#string--optimized-for-memory-preserving

I guess I could have simply put `.freeze` instead of '-'. `freeze` already does it event in 2.4

Reply:
Oh! Let's use `.freeze` until we change it everywhere?

Reply:
On it